assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -158,29 +163,29 @@ describe('Font Picker', () => {
     expect(fontsList).toBeInTheDocument();
 
     // Move down by 2
-    await act(() => {
+    act(() => {
       fireEvent.keyDown(fontsList, {
         key: 'ArrowDown',
       });
     });
-    await act(() => {
+    act(() => {
       fireEvent.keyDown(fontsList, {
         key: 'ArrowDown',
       });
     });
 
-    await act(() => {
+    act(() => {
       fireEvent.keyDown(fontsList, {
         key: 'ArrowUp',
       });
     });
 
-    await act(() => {
+    act(() => {
       fireEvent.keyDown(fontsList, { key: 'Enter' });
     });
 
-    // Moving down by 2 and back 1 up should end up with the second font: Abel.
-    expect(onChangeFn).toHaveBeenCalledWith('Abel');
+    // Moving down by 2 and back 1 up should end up with the second font: Roboto Condensed.
+    expect(onChangeFn).toHaveBeenCalledWith(availableCuratedFonts[1].name);
```

Comment:
When cross-matching the curated fonts list with the supplied sample JSON, Roboto Condensed is the second font in the list.

<img width="556" alt="Screen Shot 2020-07-28 at 3 37 01 PM" src="https://user-images.githubusercontent.com/1738349/88724865-d1465f80-d0f0-11ea-8b8d-0f77dfdc20e3.png">